eng/common/scripts/Invoke-GitHubAPI.ps1

```diff
@@ -1,4 +1,4 @@
-if ((Get-ChildItem -Path Function: | ? { $_.Name -eq "LogWarning" }).Count -eq 0) {
+if (!(Test-Path Function:LogWarning)) {
```

Comment:
Might as well remove this and always import logging.ps1 like you have done in other places.